molgenis-ui-settings/src/main/frontend/src/router/index.js

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
+import Router from 'vue-router'
+import Settings from '../components/Settings'
+
+Vue.use(Router)
+
+export default new Router({
+  routes: [
```

Comment:
Most likely want to add [history mode](https://router.vuejs.org/en/essentials/history-mode.html) , see other molgenis vue plugins